internal/DateTimeDecorator/tests/DateTimeDecorator-specs.js

```diff
@@ -7,7 +7,7 @@ describe('DateTimeDecorator', () => {
 	test('should accept an updated JavaScript Date for its value prop', () => {
 		const Picker = DateTimeDecorator({}, function PickerBase ({locale, title, value}) {
 			const minuteValue = value.getMinutes();
-			return <div locale={locale} title={title}>{minuteValue}</div>;
+			return <div locale={locale} title={title}>{minuteValue}</div>;					// eslint-disable-line react/no-unknown-property
```

Comment:
let's just solve the lint warning. remove locale destructuring and asignment to <div>

Reply:
Done